Spatial.gd

```diff
@@ -1011,6 +1022,8 @@ func _on_ReverseTrailDirection_pressed():
 
 
 func _on_ExitButton_pressed():
+	if self.unsaved_changes:
+		save_current_map_data()
```

Comment:
What happens if a current or future bug fails to call `set_unsaved_changes(true)`